src/fava/core/ingest.py

```diff
@@ -184,3 +188,28 @@ def extract(self, filename: str, importer_name: str) -> Entries:
             )
 
         return new_entries_list[0][1]
+
+
+def filepath_in_primary_imports_folder(
```

Comment:
This function needs to replace any separators in the given filname like `filepath_in_document_folder` does. Will fix that in the merge.